ledger/sandbox-classic/src/test/suite/scala/platform/sandbox/EngineModeIT.scala

```diff
@@ -101,7 +101,7 @@ class EngineModeIT
         client <- ledger.client.LedgerClient.apply(channel, ledgerClientConfiguration)
         darContent = protobuf.ByteString.copyFrom(Files.readAllBytes(darPath))
         pkgsBefore <- client.packageManagementClient.listKnownPackages()
-        _ = pkgsBefore shouldBe 'empty
+        _ = pkgsBefore shouldBe Symbol("empty")
```

Comment:
This can probably be `pkgsBefore shouldBe empty`.

Reply:
774d1e14fdf48b0076949fde10e89b30f86079e2